ionic-angular-template/src/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm, FormGroup, FormBuilder, Validators, FormControl } from '@angular/forms';
+import { AuthService } from '../../services/auth.service';
+import { Router } from '@angular/router';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit {
+  public signInForm: FormGroup;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private alertCtrl: AlertController,
+    private formBuilder: FormBuilder
+  ) { }
+
+  ngOnInit() {
+    this.signInForm = this.formBuilder.group({
+      email: new FormControl('', Validators.compose([Validators.required, Validators.email])),
```

Comment:
You can remove "Validator.compose". You can use it directly like: `email: ['', [Validators.required, Validators.email]]`
Compose is used to compose multiple validators into a single function that returns the union of the individual error maps for the provided control.